CONTRIBUTING.md

```diff
@@ -24,9 +24,7 @@ You need to be enabled for doing this.
     * `grunt version:patch` for bugfixes, new features
     * `grunt version:minor` for all breaking changes
     * `grunt version:major` NOT YET. Still need to define milestones for 1.0.0.
-* git add .
-* git commit -m "\<new version\>"
-* git push
+* git push --follow-tags
```

Comment:
@matteofigus I've updated to above git command (which should:
*Push all the refs that would be pushed without this option, and also push annotated tags in refs/tags that are missing from the remote but are pointing at commit-ish that are reachable from the refs being pushed*) which seems to be more accurate (than pushing twice). 
Don't know if it makes sense now to move it to the grunt task.